### ISSUE TITLE
CC-BY vs CC-BY-ND(-SA) license debate

### ISSUE THREAD

**Issue by baileythegreen** [baileythegreen/Chronic-Learning#7]
[This twitter thread](https://twitter.com/annemscheel/status/1312142108698898432) alleges that CC-BY is better. Do further research into this.

**Comment by thismatters** [comment]
I use a CC-BY-NC-SA for tutorpaul.com. I chose those terms because I'm well aware that a service like chegg.it would happily rip off my material and sell it to students. They may well have still done that since a license is only as strong as the legal team which enforces it, but as yet I haven't been told by any students about any infringement.

What do you think about delegating license choice to individual contributors? When they submit material they could choose the license they like (with a default suggested for those who don't care/know). The site itself could be protected by a license of our choosing, while any content is protected by the license chosen by the author.

**Comment by baileythegreen** [comment]
I picked CC-BY-NC-SA for similar reasons. But the suggestions that it might limit those working for a university that is a commercial entity from using materials are troubling.

I am not opposed to the idea of supporting a level of choice for the content creators, but maybe within limits? Like, it still needs to be something that falls under open access. So, maybe the CC-SA part is required? Idk. Things to think about.

I am pretty sure that the materials can be used under different license terms if the user negotiates that with the creator/licensor.

**Comment by thismatters** [comment]
Yeah, I saw that in the twitter thread. That would be somewhat troubling if the NC license were applied across all content; however certain content creators (myself included) won't really want their content to be used as part of university offered coursework. 

There isn't a technical barrier to allowing creators to set their own license, it is just a matter of putting another question on the form. I also know of no legal barrier to having content which is differently protected appearing on the same site or even the same page.

It is probably a good idea to think about works which might already carry a non-CC license, and how we might facilitate carrying forward that license.

**Comment by baileythegreen** [comment]
> It is probably a good idea to think about works which might already carry a non-CC license, and how we might facilitate carrying forward that license.

Do you mean like, it might have an MIT license, or an Apache one? Or do you mean something that is actually not open?

I wouldn't think the latter is applicable, so I assume you mean the former.

(It's nice having another actual person in this debate!)

**Comment by thismatters** [comment]
Yeah, I was talking about something like GPL or MIT. I don't know how common those are outside of software, but I imagine that there are other open licenses than CC.

**Comment by baileythegreen** [comment]
There are. I was kind of thinking CC-SA *or equivalent*, but I didn't say that, and I don't know the non-CC licenses very well.

**Comment by baileythegreen** [comment]
\*she says, as though she **does** know the CC licenses well\*
